wiki/Gameplay/Daily_challenge/en.md

```diff
@@ -0,0 +1,63 @@
+# Daily Challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing beatmaps consecutively with increasing difficulty.
+
+Each day introduces one new map from the [Featured Artist](/wiki/People/Featured_Artists) listing with increased difficulty. Once every 7 days however, the difficulty resets again.
+
+Sometimes, the maps come with a mod forced onto users, which requires them to pass the map with the selected mod.
+
+## Game menu
+
+From the main menu, the daily challenge menu can be accessed with the following steps:
+
+1. Click the `Play` button or press `P`.
+2. Click the `Daily challenge` button or press `D`.
+
+![](img/Daily-challenge-menu.png "Screenshot of the daily challenge menu during November 7, 2024.") 
```

Comment:
```suggestion
![](img/daily-challenge-menu.png "Screenshot of the daily challenge menu during November 7, 2024.") 
```

The image is not displayed correctly because the name doesn't match.

Reply:
It works for me (i have a plugin in VS code that allows me to preview the article in osu! wiki styling)
Will change anyway because it could really work incorrectly in other places.